README.md

```diff
@@ -1,72 +1,103 @@
 # hello-world-fwless
-Simple java rest API without framework
+Simple java API rest without framework
 
+This branch aims to showcase :
+- How to Use [Dekorate](https://github.com/dekorateio/dekorate) to generate the MANIFESTS,
+- To build using JIB the container image,
+- To deploy this microservice on an OpenShift cluster.
 
-This branch aims to showcase how to use [Dekorate](https://github.com/dekorateio/dekorate) to generate the YAML resources and next deploy this microservice on a kubernetes cluster.
-
-# OpenShift resources generation and deploy
+# OpenShift resources generation and deployment
 ## Manifests generation
-Generating these manifests (YAML files) using `Dekorate` is very easy, you just need to add the proper dependency to the `pom.xml`.
 
-```
+Generating the MANIFESTS (Openshift YAML files) using `Dekorate` is very easy as 
+you just need to add the proper dependency to the `pom.xml` .
+
+```xml
  <dependency>
       <groupId>io.dekorate</groupId>
       <artifactId>openshift-annotations</artifactId>
       <version>1.0.1</version>
  </dependency>
 ```
 
-Next, we will add a Java annotation for `Dekorate` to tune the YAML resources. It's possible [to configure Dekorate using](https://github.com/dekorateio/dekorate#usage) Java annotations, configuration properties (application.properties), both.
-In this case we will use, Java annotations, more specificlly  `@OpenshiftApplication` which also gives us access to more OpenShift specific configuration options.
-Edit the App class and add the following annotation.
+Next, you will add this Java annotation `@OpenshiftApplication` for `Dekorate` to generate the OpenShift YAML resources
+but also to tune the configuration.
 
-```
+So, edit the Java `App class` and add the following annotation.
+
+```java
 @OpenshiftApplication(
         name = "hello-world-fwless-openshift",        
         ports = @Port(name = "web", containerPort = 8080),  
         expose = true, 
         imagePullPolicy = ImagePullPolicy.Always 
 )
 ```
-- We need to prevent Dekorate that a Service should be created. A OpenShift Service is a resource providing a single, constant point of entry to our application. It has an IP address and port that never change while the service exists. Dekorate will generate a OpenShift Service in the manifest if a **`@Port`** is defined.
-- **`expose = true`** controls whether the application should be exposed via a `Route` resource accessible from the outside the cluster.
-- We use **`Always`** in order to be able to use an updated image.
-
-Trigger the manifests generation. Navigate to the directory and run `mvn clean package`. The generated manifests can be found under `target/classes/META-INF/dekorate`.
+- To tell to Dekorate that a Kubernetes `Service` should be created, then we configure the **`@Port`** parameter to specify the name of the service to be used and port.
```

Comment:
parameter -> annotation
name of the service to be used -> created
and port  -> and HTTP port